src/ControlSystem/Protocols/ControlSystem.hpp

```diff
@@ -22,6 +22,10 @@ namespace control_system::protocols {
 ///   corresponds to the name of the FunctionOfTime controlled by this
 ///   system.
 ///
+/// - a static function `component_name` returning a `std::string`.  This
+///   gives a name associated to each component of the FunctionOfTime that is
```

Comment:
A translation function of time has x, y, and z components. A quaternion rotation function of time has 4 components (1 scalar, 3 vector). A function of time that determines the shape/size of a BH has ell=0,1,2... m=0,+-1, +-2... components. Does that help?

Reply:
Yes, but it won't make the comment any clearer for the next person :) I would suggest giving the translation example to clarify, especially since this is something a user is supposed to understand.